### PR TITLE
chore: update verdaccio version and config

### PR DESCRIPTION
**Pull request by JamesHenry**
Trying to combat integration test install flakiness by implementing the suggestions from: https://github.com/verdaccio/verdaccio/discussions/2038

I needed to regenerate the README because there are open rule PRs

### CODE REVIEW

File: packages/integration-tests/local-registry/config.yml [hunk 1]
```diff
@@ -8,12 +8,18 @@ auth:
...
+    cache: false
+    maxage: 30m
   yarn:
     url: https://registry.yarnpkg.com
-    max_fails: 20
+    max_fails: 30
     fail_timeout: 10m
+    timeout: 60s
+    cache: false
+    maxage: 30m
```
Comment (juanpicado) [inline packages/integration-tests/local-registry/config.yml:22]: You missed the `keep-alive` is the one which actually boots performance.

Comment (juanpicado) [reply packages/integration-tests/local-registry/config.yml:22]: https://github.com/verdaccio/verdaccio/blob/master/conf/default.yaml#L39

**Review by juanpicado — CHANGES_REQUESTED** [review]
Once you apply `keep-alive` all performance issues should gone.

File: packages/integration-tests/local-registry/config.yml (hunk 1 above)
Comment (juanpicado) [reply packages/integration-tests/local-registry/config.yml:22]: Also this
```
logs:
  - { type: stdout, format: pretty, level: http }
  ```
slow down a lot, don't use `logs` on CI, change `http` to `warn`.

Comment (JamesHenry) [reply packages/integration-tests/local-registry/config.yml:22]: Sorry maybe I misunderstood how the config gets applied then, I had assumed default config meant I didn't have to enable it? Are you saying that by specifying any kind of uplink in my own config I am no longer getting any of the default values?

Comment (juanpicado) [reply packages/integration-tests/local-registry/config.yml:22]: Yes, currently the keep alive is not built in, you must defined it. In the next minor would be built in then you don't need that configuration anymore (only keep alive)

Comment (JamesHenry) [reply packages/integration-tests/local-registry/config.yml:26]: Ah ok thanks

Comment (JamesHenry) [reply packages/integration-tests/local-registry/config.yml:26]: And thanks I've updated the logs config

**Review by juanpicado — APPROVED** [review]
Looks like pretty green now :)

**Comment by JamesHenry** [comment]
Nice, thank you @juanpicado! I'll let you know if I observe any more issues

**Pull request merged**